src/gui_common/TweakedColourPicker.cs

```diff
@@ -109,182 +62,55 @@ public new ColorModeType ColorMode
                 ValidateRgbColor();
 
             base.ColorMode = mode;
-            UpdateButtonsState();
             UpdateTooltips();
         }
     }
 
-    /// <summary>
-    ///   Decide if user can edit the presets.
-    /// </summary>
-    [Export]
-    public bool PresetsEditable
-    {
-        get => presetsEditable;
-        set
-        {
-            presetsEditable = value;
-
-            if (addPresetButton == null)
-                return;
-
-            addPresetButton.Disabled = !value;
-        }
-    }
-
-    /// <summary>
-    ///   Decide if the presets and the add preset button is visible.
-    ///   Completely hides the native one to avoid hidden native controls reappearing.
-    /// </summary>
-    [Export]
-    public new bool PresetsVisible
-    {
-        get => presetsVisible;
-        set
-        {
-            presetsVisible = value;
-
-            if (presetsContainer == null)
-                return;
-
-            if (addPresetButton == null)
-                throw new Exception("Preset button is null even though presets container is initialized already");
-
-            separator.Visible = value;
-            presetsContainer.Visible = value;
-            addPresetButton.Visible = value;
-        }
-    }
-
     // TODO: fix this with Godot 4
     /// <summary>
     ///   Hide Godot property PresetsEnabled to avoid unexpected changes
     ///   which may cause the hidden native buttons reappear.
     /// </summary>
-    private bool PresetsEnabled { get => PresetsEditable; set => PresetsEditable = value; }
-
     public override void _Ready()
     {
         base._Ready();
 
         // Hide replaced native controls. Can't delete them because it will crash Godot.
         // TODO: fix this: this no longer has the same child layout
```

Comment:
This TODO comment was about Godot 4, so if this PR works this TODO comment needs to be removed.